test/parallel/test-tick-processor-arguments.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+require('../common');
+const tmpdir = require('../common/tmpdir');
+const fs = require('fs');
+const assert = require('assert');
+const { spawnSync } = require('child_process');
+
+tmpdir.refresh();
+process.chdir(tmpdir.path);
+
+// Generate log file.
+spawnSync(process.execPath, [ '--prof', '-p', '42' ]);
+
+const logfile = fs.readdirSync('.').filter((name) => name.endsWith('.log'))[0];
+assert(logfile);
+
+// Make sure that the --preprocess argument is passed through correctly.
+const { stdout } = spawnSync(
```

Comment:
Maybe a note here about any arguments accepted by `deps/v8/tools/tickprocessor.js` would do in case V8 removes that in the future?

Reply:
Yup, done (and also fixed CI a bit) :)